ui/client/components/modals/GenericModalDialog.tsx

```diff
@@ -29,47 +40,46 @@ class GenericModalDialog extends React.Component {
 
   onOk = () => {
     this.setState({pendingRequest: true})
-    this.props.confirm(this.closeDialog)
-      .then(this.closeDialog, this.removePending)
+    this.props.confirm(this.closeDialog).then(
+      this.closeDialog,
+      this.removePending,
+    )
   }
 
   renderOkBtn = () => {
-    return(
+    const STYLE = "zoom-in"
```

Comment:
Please declare STYLE as const in GenericModalDialog.

Reply:
gives nothing positive but separates from real one time usage. will make separating button from this component harder in future.

Reply:
Bu now it's create each time when you run renderOkBtn. When it will be as a constant class then will be declard once time.

Reply:
...and it makes no difference, but makes readability worst

Reply:
Only in your opinion?